### PR TITLE
Jpeg backend

### LINKED ISSUE #266: Jpeg backend for minimal logging
I think it would be greate to add the option to log images in jpeg format to save space.
This mostly concerns the `hooks.minimal_logging_hook.LoggingHook` class.

The simplest solution would be to pass the image format as an argument and add the an additional integrations field, such as

```
    integrations:
        minimal_logging:
            image_format: png # png or jpg. jpg saves a lot of space
```
If we keep the png format as default, nothing changes.

### PR DESCRIPTION
**Pull request by theRealSuperMario**
fixes #266 

It also moves the `git.rst` in the docs to `integrations.rst` as the content mostly talks about integrations anyway.

### CODE REVIEW

File: edflow/iterators/template_iterator.py [hunk 1]
```diff
@@ -46,8 +46,21 @@ def __init__(self, *args, **kwargs):
         self._log_ops = set_default(
             self.config, "log_ops", ["train/log_op", "validation/log_op"]
         )
+        default_minimal_logging = {
+            "active": True,
```
Comment (pesser) [inline edflow/iterators/template_iterator.py]: keys active and handlers are never used afaik. Should be either removed or made functional.

**Review by pesser — CHANGES_REQUESTED** [review]
Thanks for the feature. Besides the comment, we should merge #265 after minor changes to fix failing tests due to documentation failing.

File: edflow/iterators/template_iterator.py (hunk 1 above)
Comment (theRealSuperMario) [reply edflow/iterators/template_iterator.py]: Making it functional is going to take me a little bit I guess. When implementing the feature I have struggled to figure out how to test new functionality.

Comment (theRealSuperMario) [reply edflow/iterators/template_iterator.py]: So I just removed them

**Review by pesser — APPROVED** [review]

**Pull request merged**